docs/DeveloperGuide.md

```diff
@@ -654,11 +654,38 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 **Extensions**
 
 * 2a. User enters an invalid command
-    * 2b1. FitBook alerts the user that the command is invalid and displays the correct format
+    * 2a1. FitBook alerts the user that the command is invalid and displays the correct format
     * Use case resumes
 
 <hr>
 
+**System**: FitBook
+
+**Use case**: UC05 - Find a client
+
+**Actor**: User
+
+**MSS**
+
+1. User requests to find a client by an attribute.
+1. Fitbook displays a success message with the number of clients found, along with the filtered list of clients.
+1. Use case Ends
```

Comment:
Should we standardise having a full stop at the end of every sentence?

Reply:
Oh good catch! Fixed it!